src/Estimators/OneBodyDensityMatrices.h

```diff
@@ -276,12 +300,16 @@ class OneBodyDensityMatrices : public OperatorEstBase
 
   //  basis set updates
   void updateBasis(const Position& r, ParticleSet& pset_target);
+  //  basis set updates with spin
```

Comment:
use `///` or `/** ... */` otherwise method documentation won't be produced by doxygen.

Reply:
addressed these